.knip.json

```diff
@@ -1,5 +1,9 @@
 {
-  "ignore": ["portal/i18n/request.ts", "subgraphs/utils/index.ts"],
+  "ignore": [
+    "portal-backend/**/config.js",
+    "portal/i18n/request.ts",
+    "subgraphs/utils/index.ts"
```

Comment:
whas this line added for any particular reason? Prior to this PR, this does not throw

Reply:
See https://knip.dev/guides/working-with-commonjs

So, as you're exporting in your `config.js`

```js
module.exports = {
  get,
}
```

It expects that you import it as a named export, but you're importing as 

```js
const config = require('./src/config')
```

which is an equivalent or a `default export` ("equivalent", as there's not such thing in CommonJS)